website/docs/cloud-docs/workspaces/settings/notifications.mdx

```diff
@@ -39,24 +39,24 @@ Go to the workspace's notifications settings and press the "Add Notification" bu
-  * If you are sending notifications to Slack, see Slack's documentation on [how to create an incoming webhook](https://api.slack.com/messaging/webhooks#create_a_webhook).
+- **Webhook URL** Only available for generic, Slack, and Microsoft Teams webhooks. The destination URL to send the webhook payload to. This URL must accept HTTP or HTTPS `POST` requests, and should be able to do something useful with the chosen payload type.
+  - For Slack notifications, refer to Slack's documentation on [how to create an incoming webhook](https://api.slack.com/messaging/webhooks#create_a_webhook).
+  - For Microsoft Teams notifications, refer to Microsoft's documentation on [how to create an incoming webhook](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook).
+- **Token** Optional; only available for generic webhooks. An arbitrary secret string that Terraform Cloud will use to sign its notification webhooks. Refer to [Notification Authenticity][inpage-hmac] for details. This setting is encrypted for storage, so you cannot view it after saving.
 
-* **Token** _(optional; only available for generic webhooks):_ An arbitrary secret string, which Terraform Cloud will use to sign its notification webhooks. ([See "Notification Authenticity" below][inpage-hmac].) This setting is encrypted for storage, and cannot be viewed after it is saved.
+- **Email Recipients** Only available for emails. Select the users from the workspace's organization to notify.
 
-* **Email Recipients** _(only available for emails):_ Select the users from the workspace's organization to notify.
-
-* **Triggers:** Whether to notify this destination for all events, or only for selected events. The following specific events are available:
-  * **Created**: When a run is created and enters the "Pending" state.
-  * **Planning**: When a run acquires the lock and starts to execute.
-  * **Needs Attention**: Human decision required. When a plan has changes and is not auto-applied, or requires a policy override.
-  * **Applying**: When a run begins the apply stage, after a plan is confirmed or auto-applied.
-  * **Completed**: When a run has completed on a happy path and can't go any further.
-  * **Errored**: When a run has terminated early due to error or cancellation.
+- **Triggers:** Whether to notify this destination for all events, or only for selected events. The following specific events are available:
+  - **Created**: When a run is created and enters the "Pending" state.
+  - **Planning**: When a run acquires the lock and starts to execute.
+  - **Needs Attention**: Human decision required. When a plan has changes and is not auto-applied, or requires a policy override.
+  - **Applying**: When a run begins the apply stage, after a plan is confirmed or auto-applied.
+  - **Completed**: When a run has completed on a happy path and can't go any further.
```

Comment:
Just want to check with @tylerwolf that "successfully" is correct here, in case there is some sort of success state which is different than the "completed" state

Reply:
I think "successfully" makes sense here 👍